index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="./stylesheets/common.css">
+    <link rel="stylesheet" href="./stylesheets/mobile.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="hero">
+
+        <!-- head section -->
+
+        <header>
+            <a href="#" class="header-title">My Logo</a>
```

Comment:
- [ ] please kindly change this text from `My logo`  to your actual `Name` for identification purposes seeing that you currently do not have a logo yet.